che-terminal-client/package.json

```diff
@@ -27,18 +27,19 @@
     "src/**/*.ts"
   ],
   "devDependencies": {
+    "2": "^1.0.2",
```

Comment:
Could we use strict versions for npm libs?

Reply:
That change occurred when I ran maven to do the build -- not sure what "2" is here. It also added carats (^) to a number of other versions and reordered things a little (see below).

Reply:
It was on purpose that all versions didn't have carats.
As each version need to have a valid Eclipse CQ first, then we've to stick to an accepted version

Reply:
Per Roland Grunberg, keeper of all things Orbit, "I believe CQs apply only if the content is to be distributed by Eclipse foundation as well." 

So you're probably fine to USE and not REDISTRIBUTE these plugins in your build.